lib/node_runner.go

```diff
@@ -344,11 +268,10 @@ func (nr *NodeRunner) handleMessageFromClient(message sebaknetwork.Message) (err
 		Message:        message,
 	}
 
-	if err = sebakcommon.RunChecker(checker, nr.handleMessageFromClientCheckerDeferFunc); err != nil {
-		if _, ok := err.(sebakcommon.CheckerErrorStop); ok {
-			return
+	if err = sebakcommon.RunChecker(checker, nr.handleMessageCheckerDeferFunc); err != nil {
+		if _, ok := err.(sebakcommon.CheckerErrorStop); !ok {
+			nr.log.Error("failed to handle message from client", "error", err)
 		}
-		nr.log.Error("failed to handle message from client", "error", err)
```

Comment:
So if it's another kind of error than `CheckerErrorStop` we are not logging it ?

Reply:
@Geod24 This will `log.Error` except `sebakcommon.CheckerErrorStop`.

Reply:
Oh right I misread